afs-ws/afs-ws-server-utils/src/main/java/com/powsybl/afs/ws/server/utils/SimpleKeyGenerator.java

```diff
@@ -6,18 +6,17 @@
  */
 package com.powsybl.afs.ws.server.utils;
 
-import javax.crypto.spec.SecretKeySpec;
+import io.jsonwebtoken.Jwts;
+
 import java.security.Key;
 
 /**
  * @author Ali Tahanout {@literal <ali.tahanout at rte-france.com>}
  */
 public class SimpleKeyGenerator implements KeyGenerator {
 
-    private static final String SIMPLE_KEY = "simplekey";
-
     @Override
     public Key generateKey() {
-        return new SecretKeySpec(SIMPLE_KEY.getBytes(), 0, SIMPLE_KEY.getBytes().length, "DES");
+        return Jwts.SIG.HS512.key().build();
```

Comment:
If the old webserver is not removed:
This change of algorithm should be reported in the PR description, just in case someone is using the ws without using the corresponding client (really unlikely).

Reply:
added